### ISSUE TITLE
no 'GraphAttention' function implementation

### ISSUE THREAD

**Issue by jiyanbio** [jingraham/neurips19-graph-protein-design#2]
class SequenceModel(nn.Module):
    def __init__(self, num_letters, hidden_dim, num_layers=3,
        vocab=20, top_k=30, num_positional_embeddings=16):
        """ Graph labeling network """
        super(SequenceModel, self).__init__()

        # Hyperparameters
        self.top_k = top_k
        self.hidden_dim = hidden_dim
        self.positional_embeddings = PositionalEmbeddings(num_positional_embeddings)

        # Embedding layers
        self.W_e = nn.Linear(num_positional_embeddings, hidden_dim, bias=True)
        self.W_s = nn.Embedding(vocab, hidden_dim)

        # Decoder
        self.decoder_layers = nn.ModuleList([
            **GraphAttention(hidden_dim, hidden_dim*3)**
            for _ in range(2 * num_layers)
        ])
        self.W_out = nn.Linear(hidden_dim, num_letters, bias=True)

        # Initialization
        for p in self.parameters():
            if p.dim() > 1:
                nn.init.xavier_uniform_(p)

**Comment by jingraham** [comment]
Ah, thanks. This was due to an incomplete refactor of the module names. I started a patch [here](https://github.com/jingraham/neurips19-graph-protein-design/pull/3) that should fix it and will merge when I can test the code.
